x/axelarnet/abci.go

```diff
@@ -19,11 +19,14 @@ func BeginBlocker(_ sdk.Context, _ abci.RequestBeginBlock) {}
 // EndBlocker called every block, process inflation, update validator set.
 func EndBlocker(ctx sdk.Context, _ abci.RequestEndBlock, bk types.BaseKeeper, ibcKeeper keeper.IBCKeeper) ([]abci.ValidatorUpdate, error) {
 	queue := bk.GetIBCTransferQueue(ctx)
+	endBlockerLimit := bk.GetEndBlockerLimit(ctx)
```

Comment:
I like just use `bk.GetParams(ctx).EndBlockerLimit`, not a strong opinion

Reply:
I was doing it to be consistent with axelarnet pattern. But looking into this more, this approach will actually do more [work](https://github.com/cosmos/cosmos-sdk/blob/v0.45.6/x/params/types/subspace.go#L217) since it's loading the entire param set instead of just the relevant one.

Reply:
Good point